aio_pika/channel.py

```diff
@@ -223,13 +223,21 @@ def _on_return(self, message: aiormq.types.DeliveredMessage):
 
         return exchange
 
+    async def get_exchange(self, name: str) -> Queue:
+        """
+
```

Comment:
```
It's a shortcut for ``.declare_exchange(..., passive=True)``.
```

Reply:
Yes

Reply:
Correct. Please add it as a docstring.

Reply:
I didn’t understand. Do you want copy code from 233 line and put it to 228?

Reply:
Could you please add docstring for new methods.